tsconfig.json

```diff
@@ -6,18 +6,14 @@
         "noImplicitAny": true,
         "moduleResolution": "node",
         "sourceMap": true,
+        "declaration": true,
+        "declarationMap": true,
         "outDir": "dist",
         "baseUrl": ".",
-        "allowJs": true,
```

Comment:
There aren't any `.js` files being transpiled by TypeScript, so this can be removed.